### PR TITLE
Go timeouts

### PR DESCRIPTION
**Pull request by domudall**
Checks to make sure default `http` library that have no default timeouts are not used.

### CODE REVIEW

File: ci.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 ___author___ = "Jim Hill (github.com/jimah)"
 ___credits___ = ["Jim Hill (github.com/jimah)",
                  "Lee Archer (github.com/lbn)",
+                 "Dom Udall (github.com/domudall)",
```
Comment (domudall) [inline ci.py]: 🌚

File: ci.py [hunk 2]
```diff
@@ -188,6 +189,66 @@ def go_lint(package):
...
+            file_pattern = re.compile('^([a-zA-Z0-9\/]+.go)\:([0-9]+)')
+
+            for line in lines:
+                filename_match = re.match(file_pattern, line)
+
+                if filename_match is None:
+                    continue
+
+                err = True
+                output += filename_match.group(1)
```
Comment (jimah) [inline ci.py]: Maybe seperate this into a function?

e.g `get_error_message` nested within the go_timeouts func?

Comment (domudall) [inline ci.py]: Done

**Comment by jimah** [comment]
🉑

**Pull request merged**